package.json

```diff
@@ -20,6 +20,15 @@
     "meteor-client": "^1.2.0"
   },
   "devDependencies": {
-    "browserify": "^11.2.0"
+    "browserify": "^11.2.0",
+    "gulp": "^3.9.0",
```

Comment:
Since `gulp` is a dependency, why should the user install it globally? They could do a command:

~~~
./node_modules/bin/gulp
~~~

Or simply you could make a command:

~~~
npm run build
~~~

This is more universal since all node applications can have commands, but not all contain gulp.